src/RedHat.AspNetCore.Server.Kestrel.Transport.Linux/Transport.cs

```diff
@@ -25,21 +26,19 @@ private enum State
         // Kestrel LibuvConstants.ListenBacklog
         private const int ListenBacklog = 128;
 
-        private readonly IEndPointInformation _endPoint;
-        private readonly IConnectionDispatcher _connectionDispatcher;
         private readonly LinuxTransportOptions _transportOptions;
         private readonly ILoggerFactory _loggerFactory;
         private readonly ILogger _logger;
         private State _state;
         private readonly object _gate = new object();
         private ITransportActionHandler[] _threads;
+        private TransportThread[] _transportThreads;
```

Comment:
Can you remove `_transportThreads` by adding `ValueTask<TSocket> AcceptAsync(CancellationToken cancellationToken = default)` to `ITransportActionHandler`?